lib/page/house_wikidata.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+require_relative '../popolo_helper'
```

Comment:
why did you need to add this? I can't see anything that should require it, and removing it doesn't cause any obvious failure.